lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MojoParameter.java

```diff
@@ -0,0 +1,158 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Red Hat Inc. and others.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ * Contributors:
+ *      Anton Tanasenko. - initial API and implementation (in m2e)
+ *******************************************************************************/
+package org.eclipse.lemminx.maven;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+// Copied from m2e
```

Comment:
I'd put a note about it in the copyright header "Contributors" section, adding `Andrw Obuchowicz - copy from m2e to lemminx-maven`

Reply:
Sounds good